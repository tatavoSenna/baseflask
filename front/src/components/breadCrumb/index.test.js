import React from 'react'
import { render } from '@testing-library/react'

import Footer from './index.js'

describe('<Footer />', () => {
	const currentTest = 'Current Test'
	it('Snapshot testing', () => {
		const { asFragment } = render(<Footer current={currentTest} />)

		expect(asFragment()).toMatchSnapshot()
	})

	it('Testing props:', () => {
		const { getByTestId, getByText } = render(<Footer current={currentTest} />)

		expect(getByTestId('current')).toContainElement(getByText('Current Test'))
	})
})
