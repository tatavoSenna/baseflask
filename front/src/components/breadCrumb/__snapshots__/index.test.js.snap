// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="ant-breadcrumb"
    style="margin: 16px 30px;"
  >
    <span>
      <span
        class="ant-breadcrumb-link"
        data-testid="current"
      >
        Current Test
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
  </div>
</DocumentFragment>
`;
